main/seccomp.c

```diff
@@ -0,0 +1,60 @@
+#include "seccomp-bpf.h"
+#include <signal.h>
+#include <stdio.h>
+#include <stddef.h>
+#include <string.h>
+#include <sys/prctl.h>
```

Comment:
I redid this based on libseccomp, which also gets rid of the LICENSE file requirement.